tools/s2i-dpdk/Dockerfile

```diff
@@ -67,6 +67,9 @@ COPY ./s2i/bin/ /usr/libexec/s2i
 
 RUN setcap cap_sys_resource,cap_ipc_lock=+ep /usr/libexec/s2i/run
 
+# Allows non-root users to use dpdk-testpmd.
+RUN setcap cap_sys_resource,cap_ipc_lock,cap_net_raw+ep /usr/local/bin/dpdk-testpmd
```

Comment:
should it have `=`

Reply:
also can you add all the other dpdk-* commands?

Reply:
it's not needed the =
will add it to the other commands

Reply:
Had another thought about this. 

I think it's better to add the caps to the other commands when needed. I was blindly adding it for all of them but then I realized that a command like "dpdk-procinfo" doesn't need to allocate memory but needs to able to set CPU affinitiy. For that, it needs the CAP_SYS_NICE capability. So the requirements are different for some of them.

Note that the downstream image (gitlab) only has dpdk-testpmd and some python scripts. The other commands are not there.

Reply:
so lets do it for the stuff that we use

```
dpdk-testpmd
dpdk-testbbdev
```

lets also add a comment so we don't forget about it if in the future we need to support other programs

Reply:
where do we use dpdk-testbbdev?

Reply:
@SchSeba, I can't find a test that uses it in this repo

Reply:
we use it on d/s

Reply:
what do you exactly mean with d/s?

Reply:
(discussed internally with @SchSeba, testbbdev in used in cnf-gotests). Thanks @SchSeba, I added the caps.